spec/requests/expense_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Expense, type: :request do
+  include Devise::Test::IntegrationHelpers
+  before :each do
+    @user = User.create(name: 'Paulina', email: 'paulina@gmail.com', password: '123456')
+    @expense = Expense.create(name: 'Gym', amount: 400, user: @user)
+    @group = Group.create(name: 'Health', icon: 'hospital.png', user: @user)
+    @group_expense = GroupExpense.create(group: @group, expense: @expense)
+    sign_in @user
+  end
+
+  describe 'GET /index' do
+    before :each do
+      get group_path(@group.id)
+    end
+
+    it 'returns a successful status' do
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly you have failed `spec`, please fix it, all `specs` should be passed 🙏🏼 

![image](https://github.com/adarachel/budgetapp/assets/63915024/0569a0ed-a40c-48d1-9514-63c34720974e)

Reply:
Hello reviewer @mukhammadrizooff Thank you for your review. However, my specs are all passing in my terminal as you can see in the attached screenshot below, i made sure they were all passing before requesting a review:

![notestfailures](https://github.com/adarachel/budgetapp/assets/108772486/383e5615-85f9-4687-a36c-0b5b4484c422)

Reply:
As you can see again, the test spec file expense_spec.rb  seems to be passing fine:

![testspassing](https://github.com/adarachel/budgetapp/assets/108772486/263855a5-966c-4884-bf4c-6abfdcae352a)